src/Sulu/Bundle/MediaBundle/Entity/BaseCollection.php

```diff
@@ -122,16 +123,6 @@ public function getCreator()
     }
 
     /**
-     * Constructor.
-     */
-    public function __construct()
-    {
-        $this->meta = new \Doctrine\Common\Collections\ArrayCollection();
-        $this->media = new \Doctrine\Common\Collections\ArrayCollection();
-        $this->children = new \Doctrine\Common\Collections\ArrayCollection();
```

Comment:
Wouldn't it be better to let these initializations in this class? The it would work for all child classes (assuming they are using `parent::__construct()`)